src/index.css

```
@@ -109,6 +140,6 @@ All colors MUST be HSL.
 
   body {
     @apply bg-background text-foreground;
-    font-family: 'Noto Serif Devanagari', serif;
+    font-family: 'Vesper Libre', serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)